<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeValueEnumOptionsWidget</class>
 <widget class="QWidget" name="NodeValueEnumOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>29</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="ValueEdit"/>
   </item>
   <item>
    <widget class="QComboBox" name="OptionsCombo"/>
   </item>
   <item>
    <widget class="CheckIndicator" name="ValueValidIndicator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckIndicator</class>
   <extends>QLabel</extends>
   <header>gui/widgets/checkindicator.hui</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections/>
</ui>
